### PR TITLE
 clean up scripts in #2572

### PR DESCRIPTION
**Pull request by xiaohui-zhang**

### CODE REVIEW

**Comment by yangxueruivs** [comment]
Hi, I noticed that you didn't add a recipe for librispeech, is it because it won't give any improvement in that dataset? I tried myself and found it's not as good as pure tdnn-f.

**Comment by danpovey** [comment]
We didn't run it on Librispeech.  When you say you ran it on librispeech, can you be more specific about the configuration you ran?  Because we'd tend to use slightly larger models for librispeech, and other settings like num-epochs may be different there too.

**Comment by yangxueruivs** [comment]
@danpovey Sorry, the configurations are:

`input dim=100 name=ivector`
`input dim=40 name=input`

`  # MFCC to filterbank`
`  idct-layer name=idct input=input dim=40 cepstral-lifter=22 affine-transform-file=$dir/configs/idct.mat
`
`  linear-component name=ivector-linear $ivector_affine_opts dim=200 input=ReplaceIndex(ivector, t, 0)`
`  batchnorm-component name=ivector-batchnorm target-rms=0.025`
`  batchnorm-component name=idct-batchnorm input=idct`

`  combine-feature-maps-layer name=combine_inputs input=Append(idct-batchnorm, ivector-batchnorm) num-filters1=1 num-filters2=5 height=40`
`  conv-relu-batchnorm-layer name=cnn1 $cnn_opts height-in=40 height-out=40 time-offsets=-1,0,1 height-offsets=-1,0,1 num-filters-out=64`
`  conv-relu-batchnorm-layer name=cnn2 $cnn_opts height-in=40 height-out=40 time-offsets=-1,0,1 height-offsets=-1,0,1 num-filters-out=64`
`  conv-relu-batchnorm-layer name=cnn3 $cnn_opts height-in=40 height-out=20 height-subsample-out=2 time-offsets=-1,0,1 height-offsets=-1,0,1 num-filters-out=128`
`  conv-relu-batchnorm-layer name=cnn4 $cnn_opts height-in=20 height-out=20 time-offsets=-1,0,1 height-offsets=-1,0,1 num-filters-out=128`
`  conv-relu-batchnorm-layer name=cnn5 $cnn_opts height-in=20 height-out=10 height-subsample-out=2 time-offsets=-1,0,1 height-offsets=-1,0,1 num-filters-out=256`
`  conv-relu-batchnorm-layer name=cnn6 $cnn_opts height-in=10 height-out=10 time-offsets=-1,0,1 height-offsets=-1,0,1 num-filters-out=256`

`  # the first TDNN-F layer has no bypass`
`  tdnnf-layer name=tdnnf7 $tdnnf_first_opts dim=1536 bottleneck-dim=256 time-stride=0`
`  tdnnf-layer name=tdnnf8 $tdnnf_opts dim=1536 bottleneck-dim=160 time-stride=3`
`  tdnnf-layer name=tdnnf9 $tdnnf_opts dim=1536 bottleneck-dim=160 time-stride=3`
`  tdnnf-layer name=tdnnf10 $tdnnf_opts dim=1536 bottleneck-dim=160 time-stride=3`
`  tdnnf-layer name=tdnnf11 $tdnnf_opts dim=1536 bottleneck-dim=160 time-stride=3`
`  tdnnf-layer name=tdnnf12 $tdnnf_opts dim=1536 bottleneck-dim=160 time-stride=3`
`  tdnnf-layer name=tdnnf13 $tdnnf_opts dim=1536 bottleneck-dim=160 time-stride=3`
`  tdnnf-layer name=tdnnf14 $tdnnf_opts dim=1536 bottleneck-dim=160 time-stride=3`
`  tdnnf-layer name=tdnnf15 $tdnnf_opts dim=1536 bottleneck-dim=160 time-stride=3`
`  tdnnf-layer name=tdnnf16 $tdnnf_opts dim=1536 bottleneck-dim=160 time-stride=3`
`  tdnnf-layer name=tdnnf17 $tdnnf_opts dim=1536 bottleneck-dim=160 time-stride=3`
`  tdnnf-layer name=tdnnf18 $tdnnf_opts dim=1536 bottleneck-dim=160 time-stride=3`
`  tdnnf-layer name=tdnnf19 $tdnnf_opts dim=1536 bottleneck-dim=160 time-stride=3`
`  linear-component name=prefinal-l dim=256 $linear_opts`

`  prefinal-layer name=prefinal-chain input=prefinal-l $prefinal_opts big-dim=1536 small-dim=256`
`  output-layer name=output include-log-softmax=false dim=$num_targets $output_opts`

`  prefinal-layer name=prefinal-xent input=prefinal-l $prefinal_opts big-dim=1536 small-dim=256`
`  output-layer name=output-xent dim=$num_targets learning-rate-factor=$learning_rate_factor $output_opts`

different tdnnf layers are tried(17/18/19), bottleneck dims(256/160). I set the epochs always equals to 4 so that I could compare the loss with pure tdnnf.

And the relative loss is like(first column is tdnnf, the others are cnn-tdnnf):

2.95 | 3.16 | 3.18
2.94 | 3.14 | 3.18
2.93 | 3.13 | 3.17
2.93 | 3.13 | 3.16
2.94 | 3.12 | 3.14
2.92 | 3.12 | 3.13
2.92 | 3.12 | 3.07
2.89 | 3.11 | 3.1
2.91 | 3.12 | 3.1
2.92 | 3.12 | 3.1
2.89 | 3.12 | 3.08
2.9 | 3.09 | 3.08
2.86 | 3.08 | 3.06
2.88 | 3.07 | 3.05
2.85 | 3.07 | 3.05
2.9 | 3.09 | 3.06
2.88 | 3.05 | 3.04
2.85 | 3.08 | 3.06
2.85 | 3.05 | 3.03
2.83 | 3.03 | 3.07
2.83 | 3.05 | 3.02

**Comment by danpovey** [comment]
Don't focus on that one test condition.  The compare_wer.sh script compares
about 16 different test conditions.  Is there degradation consistently
across those-- or at least, on average?
The configuration seems reasonable.
